### DISCUSSION TITLE
Question: ¿How can I create my own "Downscroll" unused "Kade Engine"?

### DISCUSSION THREAD

**Discussion by PROESSPESS** [FunkinCrew/Funkin#1546] [Q&A] [answered]
How can I create my own "Downscroll" unused "Kade Engine"

**Comment by VovaGamesChips** [comment, marked as answer]
Downscroll it's part of Week 7 update, so when week 7 source code will be published you can do it without coding.
(Btw I don't know, but I had tried. Sorry.)
